examples/http.rs

```diff
@@ -4,6 +4,7 @@ extern crate cookie_factory;
 use crate::implementation::*;
 
 fn main() {
+    use cookie_factory::gen;
```

Comment:
cosmetics, but indent looke weird.

Reply:
Fixed, but can we get a `rustfmt.toml` with the code style you want to have? It's clearly not the default `rustfmt` style, and it's boring to manually adhere to some undefined code style :)